volume/volume_propagation_linux_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"testing"
 )
 
-func TestParseMountSpecPropagation(t *testing.T) {
+func TestParseMountRawPropagation(t *testing.T) {
```

Comment:
There should probably be an equivalent Windows test as the engine should also reject these.